### DISCUSSION TITLE
How to delete?

### DISCUSSION THREAD

**Discussion by NicNem55** [Chia-Network/chia-blockchain#2677] [Q&A]
how to delete an unnecessary sowing?
button is not active
https://prnt.sc/1212mlj

**Comment by climba03003** [comment]
Restart the GUI instance.

**Reply by NicNem55** [reply]
then everything is removed,
and I need some

**Reply by climba03003** [reply]
Then, let it be. Currently, no other way to remove it without restarting the GUI instance.
https://github.com/Chia-Network/chia-blockchain/releases/tag/1.1.0
`We have temporarily disabled the "Delete Plots" button in the Windows GUI as we are still working on debugging upstream issues that are causing it.`
